Invalid memory address or nil pointer dereference when collect-gentxs

I am doing Interchain-Test for Migaloo and when testing `collect-gentxs` this err occur
```
panic: runtime error: invalid memory address or nil pointer dereference
                                [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1f34926]
                            
                                goroutine 1 [running]:
                                github.com/cosmos/cosmos-sdk/x/genutil/types.ValidateAndGetGenTx({0xc000ac6500, 0x4a2, 0x4a3}, 0x298f960?, 0x0)
                                        github.com/cosmos/cosmos-sdk@v0.47.2/x/genutil/types/genesis_state.go:130 +0x106
                                github.com/cosmos/cosmos-sdk/x/genutil.CollectTxs({0x3a1e120, 0xc0007ac0b0}, 0x1500000?, {0xc000d8e920, 0x20}, {0xc00032b200, 0x28}, {{0x3addddfa, 0xedc0a9727, 0x0}, ...}, ...)
                                        github.com/cosmos/cosmos-sdk@v0.47.2/x/genutil/collect.go:114 +0x45e
                                github.com/cosmos/cosmos-sdk/x/genutil.GenAppStateFromConfig({0x3a1e120, 0xc0007ac0b0}, {0x7f50e2e53fd0, 0xc000975e40}, 0xc000634dc0, {{0xc000131f30, 0x9}, {0xc00032b200, 0x28}, {0xc00032b020, ...}, ...}, ...)
                                        github.com/cosmos/cosmos-sdk@v0.47.2/x/genutil/collect.go:32 +0x174
                                github.com/cosmos/cosmos-sdk/x/genutil/client/cli.CollectGenTxsCmd.func1(0xc000da2f00?, {0xc000289060?, 0x0?, 0x2?})
                                        github.com/cosmos/cosmos-sdk@v0.47.2/x/genutil/client/cli/collect.go:53 +0x573
                                github.com/spf13/cobra.(*Command).execute(0xc000da2f00, {0xc000289020, 0x2, 0x2})
                                        github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
                                github.com/spf13/cobra.(*Command).ExecuteC(0xc000da2900)
                                        github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
                                github.com/spf13/cobra.(*Command).Execute(...)
                                        github.com/spf13/cobra@v1.7.0/command.go:992
                                github.com/spf13/cobra.(*Command).ExecuteContext(...)
                                        github.com/spf13/cobra@v1.7.0/command.go:985
                                github.com/cosmos/cosmos-sdk/server/cmd.Execute(0x22561f0?, {0x0, 0x0}, {0xc0001adb18, 0x14})
                                        github.com/cosmos/cosmos-sdk@v0.47.2/server/cmd/execute.go:32 +0x179
                                main.main()
                                        github.com/White-Whale-Defi-Platform/migaloo-chain/v3/cmd/migalood/main.go:14 +0x30
```

--- a/app/app.go
+++ b/app/app.go
@@ -208,7 +208,7 @@ var (
 	ModuleBasics = module.NewBasicManager(
 		auth.AppModuleBasic{},
 		alliancemodule.AppModuleBasic{},
-		genutil.AppModuleBasic{},
+		genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
 		bank.AppModule{},
 		capability.AppModuleBasic{},
 		consensus.AppModuleBasic{},

